File: guess.java
import java.util.Scanner;

public class guess {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int num = Math.floor(Math.random() * 99) + 1;
        int answer;
        int tries = 0;
        while (True) {
            System.out.println("Guess a number between 1 and 100");
            answer = scan.nextInt();
            tries++;
            if (answer > num) {
                System.out.println("Too high!");
            } else if (answer < num) {
                System.out.println("too low!");
            } else {
                System.out.println("You got it! It took you " + tries + " tries!");
                break;
            }
        }
        scan.close();
    }
}

File: guess.py
#!/usr/bin/env python3

from random import randrange

number = randrange(1, 100)
tries = 0

while True:
    guess = int(input("Guess a number between 1 and 100: "))
    tries += 1
    if guess > number:
        print("Too high!")
    elif guess < number:
        print("Too low!")
    else:
        print(f"You got it in {tries} tries!")
        break